movement/movement_custom_signal_tunes.h

```diff
@@ -84,4 +84,21 @@ int8_t signal_tune[] = {
 };
 #endif // SIGNAL_TUNE_KIM_POSSIBLE
 
+#ifdef SIGNAL_TUNE_LAYLA
+int8_t signal_tune[] = {
+    BUZZER_NOTE_A6, 4,
+    BUZZER_NOTE_REST, 1,
+    BUZZER_NOTE_C7, 5,  
+    BUZZER_NOTE_REST, 2,
```

Comment:
Any reason why some rests have duration of 2, while others have 1 (and also the first note is shorter than the rest)? At least on paper, all note durations are actually same.

Reply:
I tried tweaking timings with my ear, last night reducing first note and dividing some notes based on timing sounded closer to my ear -- never trust biological organs 😓